pkg/executor/importer/import.go

```diff
@@ -931,14 +931,34 @@ func (e *LoadDataController) InitDataStore(ctx context.Context) error {
 			return exeerrors.ErrLoadDataInvalidURI.GenWithStackByArgs(target,
 				err3.Error())
 		}
-		s, err = e.initExternalStore(ctx, cloudStorageURL, target)
+		s, err = e.initSelfManagedExternalStore(ctx, cloudStorageURL, target)
 		if err != nil {
 			return err
 		}
 		e.GlobalSortStore = s
 	}
 	return nil
 }
+
+func (*LoadDataController) initSelfManagedExternalStore(ctx context.Context, u *url.URL, target string) (storage.ExternalStorage, error) {
```

Comment:
what you mean by `self-managed`?

Reply:
s3 does not belong to user, it belongs to our cloud infra